### ISSUE TITLE
SelectAll with database schema

### ISSUE THREAD

**Issue by lovis-ff** [koskimas/kysely#131]
First of all, thanks a lot for this awesome library. I have been playing with it a lot lately and am absolutely amazed.
I am working with a postgresdb in a schema different then the public one which has lead to the following issue:
I have a query like this:
```
db
  .selectFrom("my_schema.my_table")
  .selectAll("my_schema.my_table")
```

which results in a query like this `select "my_schema.my_table".* from "my_schema"."my_table"`, which results in an `missing FROM-clause entry for table "my_schema.my_table"` error.
The part in the `selectFrom` gets quoted correctly, while the part in the `selectAll` unfortunately not. The expected resulting query would be `select "my_schema"."my_table".* from "my_schema"."my_table"`.

Specifying the specific fields with the `select` function works well:
```
await tx
    .selectFrom("my_schema.my_table")
    .select("my_schema.my_table.id")
```
results in ` select "my_schema"."my_table"."id" from "my_schema"."my_table"`.

Apart from this, everything regarding other schemas than the public one works quite fine so far, thanks a lot!

**Comment by koskimas** [comment]
Thanks for reporting this! I'll fix it asap. I'm on vacation right now and don't have my laptop, so you'll have to wait until next week.

**Comment by koskimas** [comment]
Fixed in 0.20.1
